src/Entity/AgendaBroadcast.php

```diff
@@ -42,6 +43,12 @@ class AgendaBroadcast
      */
     private $agenda;
 
+    /**
+     * @ORM\Column(type="json")
+     * @Groups({"mail_template"})
+     */
+    private $customData = [];
+
```

Comment:
Consider putting this (along with the getter and setter) into a trait to reduce code duplication.

Reply:
Added a `CustomDataTrait` and used it in `HearingPostRequest`, `InspectionLetter` and `AgendaBroadcast`.